source/documentation/reference/operational-processes.md

```diff
@@ -0,0 +1,174 @@
+
+One engineer should be available to answer questions throughout the hours of support (10AM - 5PM).  
+
+### Communication style
+
+In our responses we aim to be:
+* friendly
+* timely
+* open - no question is stupid
+
+When the solution is a quick one, it is good to have the whole conversation in the channel rather than in a direct message so that other people can search/see what the resolution was..
+
+### What we communicate
+
+The main purpose of [`#ask-cloud-platform`](https://mojdt.slack.com/messages/C57UPMZLY/) is to discuss the problems that people are having and help them to solve them.
+
+### User support tickets
+
+If someone is asking for help that will be quick to do (less than 15 mins) or is mainly advice then keep the interaction in channel and get it done.
+
+If someone needs something that takes longer, is more challenging to complete or you find you've spent longer than 15 mins on it, then continue to talk in the channel but also ask the person asking for help to raise a ticket using the GitHub issue link: [long version](https://github.com/ministryofjustice/cloud-platform/issues/new?template=cloud-platform-support-request.md&labels=support%20team) and [short version](https://goo.gl/msfGiS).
+
+The purpose of the ticket is to keep a record of the work we are doing and how it is progressing. It is *not* the primary communication channel with the person who raised the problem - this should remain slack where you can provide a richer and more human interaction, answering questions if necessary. The engineer working on the ticket should update the ticket for our record as work progresses.
+
+In many cases it can be more helpful for the engineer on support to create the ticket rather than the person reporting the problem as you will be able to provide more relevant context.
+
+Once they have created the ticket it will appear in the `Support New Issues` column of our [sprint board](https://app.zenhub.com/workspaces/cloud-platform-team-5ccb0b8a81f66118c983c189/board?repos=126507503,131595384,122635841,175829912). It can then be moved into the relevant columns as you work on it.
+
+## `#cloud-platform-update` slack channel
+
+[`#cloud-platform-update`](https://mojdt.slack.com/messages/CH6D099DF/) has been created to make broadcast communications, examples include:
+
+* when there is planned work on a service (e.g. we are completing maintenance on Postgres RDS instance in prod)
+* when there is an incident in progress (e.g there is a problem with Jenkins, no one can access it right now)
+
+In these situations:
+
+* try not to use notifiers like `@here` and `@channel` unless absolutely necessary
+* perhaps use visual clues in your message to show it is an announcement
+* commit to a time when you will update on the situation and keep that commitment
+
+So for example:
+
+```
```

Comment:
This looks a bit weird using backticks to format it as a code block. I'd suggest using a blockquote instead, with >

<img width="863" alt="Screen Shot 2019-06-26 at 16 53 08" src="https://user-images.githubusercontent.com/2338/60195393-e89e8380-9832-11e9-92c2-1dbc00eeb9a2.png">

Reply:
changed